browser-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "name": "browser-app",
+  "version": "0.0.1",
+  "dependencies": {
+    "@theia/core": "latest",
+    "@theia/filesystem": "latest",
+    "@theia/workspace": "latest",
+    "@theia/preferences": "latest",
+    "@theia/navigator": "latest",
+    "@theia/process": "latest",
+    "@theia/terminal": "latest",
+    "@theia/editor": "latest",
+    "@theia/languages": "latest",
+    "@theia/markers": "latest",
+    "@theia/monaco": "latest",
+    "@theia/typescript": "latest",
+    "@theia/messages": "latest",
+    "theia-machines-extension": "0.0.1"
+  },
+  "devDependencies": {
+    "@theia/cli": "latest"
```

Comment:
Set up, please, concrete version of dependencies 0.3.8 for example.

Reply:
For now, only 0.3.7 is allowed.
I've just created [CQ](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16031) for 0.3.8

Reply:
done